src/containers/App/index.test.js

```diff
@@ -33,6 +59,21 @@ describe('<App />', () => {
     expect(wrapper.find(Footer).length).toBe(1);
   });
 
+  describe('routing', () => {
+    it('can navigate to kto form', () => {
```

Comment:
Shouldn't this test verify that a click on a link loads a page that contains the `KtoContainer` component?